bin/summariser.py

```diff
@@ -171,7 +172,9 @@ def runprocess(db_config_file, config_file):
             log.warning("The summariser may not start again until it is removed.")
 
         log.info(LOG_BREAK)
-
+        db.db.close()
```

Comment:
Is `db.db` correct? Guess it doesn't help that the local name is the same as what the MySQL module uses in its hierarchy.